todo-app/aws/3-tier-app/modules/network/variables.tf

```diff
@@ -75,4 +111,45 @@ variable "instance_tenancy" {
     condition     = contains(["default", "dedicated", "host"], var.instance_tenancy)
     error_message = "Instnace tenancy either default or (dedicated or host)."
   }
-}
\ No newline at end of file
+}
+variable "env_cidr_block" {
+  type        = bool
+  description = "Add current deployment enviornment cidr block"
+  default     = true
+}
+
+
+variable "instance_count" {
```

Comment:
change variable name `bastion_instance_count` and `default=null` & add description

Reply:
done